ziti/sdkinfo/host_unix.go

```diff
@@ -0,0 +1,32 @@
+// +build !windows
+
+/*
+	Copyright 2019 NetFoundry, Inc.
```

Comment:
2020 - there's a few other spots to update from 2019

Reply:
fixed